butterfree/configs/db/kafka_config.py

```diff
@@ -0,0 +1,131 @@
+"""Holds configurations to read and write with Spark to Kafka."""
+from typing import Dict, List
+
+from butterfree.configs import environment
+from butterfree.configs.db import AbstractWriteConfig
+
+
+class KafkaConfig(AbstractWriteConfig):
+    """Configuration for Spark to connect to Kafka.
+
+    Attributes:
+        kafka_connection_string: kafka topic name.
+        mode: write mode for Spark.
+        format_: write format for Spark.
+        stream_processing_time: processing time interval for streaming jobs.
+        stream_output_mode: specify the mode from writing streaming data.
+        stream_checkpoint_path: path on S3 to save checkpoints for the stream job.
+
+    More information about processing_time, output_mode and checkpoint_path
+    can be found in Spark documentation:
+    [here](https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html)
+
+    """
+
+    def __init__(
+        self,
+        kafka_connection_string: str = None,
+        mode: str = None,
+        format_: str = None,
+        stream_processing_time: str = None,
+        stream_output_mode: str = None,
+        stream_checkpoint_path: str = None,
+    ):
+        self.kafka_connection_string = kafka_connection_string
+        self.mode = mode
+        self.format_ = format_
+        self.stream_processing_time = stream_processing_time
+        self.stream_output_mode = stream_output_mode
+        self.stream_checkpoint_path = stream_checkpoint_path
+
+    @property
+    def kafka_connection_string(self) -> str:
+        """Username used in connection to Cassandra DB."""
+        return self.__kafka_connection_string
+
+    @kafka_connection_string.setter
+    def kafka_connection_string(self, value: str):
+        input_value = value or environment.get_variable("KAFKA_CONNECTION_STRING")
```

Comment:
You can use the env variable `KAFKA_CONSUMER_CONNECTION_STRING`
```suggestion
        input_value = value or environment.get_variable("KAFKA_CONSUMER_CONNECTION_STRING")
```